static/js/edges.common.js

```diff
@@ -1487,31 +1487,31 @@ edges.mex.renderers.SelectedFilters = class extends edges.Renderer {
                 if ($.inArray(field, this.hideValues) > -1) {
                     valDisplay = "";
                 }
-                filters += `<span class="${valClass}">${val.display}</span>`;
+                filters += `<span class="${valClass}">${val.display}</span></span>`;
 
                 // the remove block looks different, depending on the kind of filter to remove
                 if (this.allowRemove) {
                     if (def.filter === "term" || def.filter === "terms") {
-                        filters += `<button class="${removeClass} img-button" data-bool="must" data-filter="${def.filter}" data-field="${field}" data-value="${val.val}" title="Remove" href="#">
-                                        <img src="/static/images/close.svg" alt="Remove" title="Remove" style="width:24px;height:24px;vertical-align:middle"/>
+                        filters += `<button class="${removeClass} img-button" aria-label='{{_("Remove")}}' data-bool="must" data-filter="${def.filter}" data-field="${field}" data-value="${val.val}" title="Remove">
+                                        <img src="/static/images/close.svg" class="img-button-icon" title="Remove"/>
                                     </button>`;
                     } else if (def.filter === "range") {
                         var from = val.from ? ' data-' + val.fromType + '="' + val.from + '" ' : "";
                         var to = val.to ? ' data-' + val.toType + '="' + val.to + '" ' : "";
-                        filters += `<button class="${removeClass} img-button" data-bool="must" data-filter="${def.filter}" data-field="${field}" ${from} ${to} title="Remove" href="#">
-                                        <img src="/static/images/close.svg" alt="Remove" title="Remove" style="width:24px;height:24px;vertical-align:middle"/>
+                        filters += `<button class="${removeClass} img-button" aria-label='{{_("Remove")}}' data-bool="must" data-filter="${def.filter}" data-field="${field}" ${from} ${to} title="Remove">
+                                        <img src="/static/images/close.svg" class="img-button-icon"  title="Remove"/>
```

Comment:
We should keep this 'alt'. Title on the button is sufficient in this case.
Read here: https://www.w3schools.com/tags/att_img_alt.asp
and here: https://stackoverflow.com/questions/872389/html-img-tag-title-attribute-vs-alt-attribute